recognition/src/ransac_based/obj_rec_ransac.cpp

```diff
@@ -42,180 +42,195 @@
 
 using namespace pcl::common;
 
-pcl::recognition::ObjRecRANSAC::ObjRecRANSAC (float pair_width, float voxel_size)
```

Comment:
Please undo all the (formatting) changes in this file, let's keep the PR on topic

Reply:
It's a mystery to me as to what changes `clang-format` will make to particular files in this code base.